packages/prop-house-wrapper/src/index.ts

```diff
@@ -158,31 +158,30 @@ export class PropHouseWrapper {
   }
 
   async postFile(file: File, name: string) {
-    if (!this.signer) return;
     try {
       const form = new FormData();
       form.append('file', file, name);
       form.append('name', name);
       const fileBuffer = Buffer.from(await file.arrayBuffer());
-      const signature = await this.signer.signMessage(fileBuffer);
-      form.append('signature', signature);
-      console.log(form);
+      if (this.signer) {
+        const signature = await this.signer.signMessage(fileBuffer);
+        form.append('signature', signature);
+      }
```

Comment:
The UX of signing a file buffer seems a bit awkward, especially if uploading multiple files. Maybe combining strict file upload rate-limiting and file size limits is enough?

Reply:
Agreed, this PR removes that signing requirement. Clients can just POST a file without signing it like you would with a file upload form. I left this in case a client wants to implement signing.

Reply:
In that case, how about an optional `requestSignature = false` param so the frontend can use the same wrapper instance without accidentally asking for a signature on file upload?

Reply:
Added to both the file and buffer functions.